test/unit/testApplicantNameAsOnWill.js

```diff
@@ -28,4 +103,69 @@ describe('ApplicantNameAsOnWill', () => {
             done();
         });
     });
+
+        describe('handlePost()', () => {
```

Comment:
These `handlePost()` tests appear to be a duplicate of the ones further up in the file?